src/components/ProjectDelete/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { Fragment, Component } from 'react';
+import { Button, Icon, Modal } from 'semantic-ui-react';
+import { IconBtn } from '../../styles/ContentsTable';
+
+// This component inmplementation is not finished yet
+// TODO : Add Delete Mutation to handle delete project
```

Comment:
I created this component but the Project Delete feature is not implemented yet.
I will implement in next pull request.